packages/docs/src/routes/demo/qwikcity/middleware/layout.tsx

```diff
@@ -8,8 +8,7 @@ export const onRequest: RequestHandler = async ({ next }) => {
       error &&
       typeof error === 'object' &&
       'message' in error &&
-      typeof error.message === 'string' &&
-      error.message == 'ERROR: Demonstration of an error response.'
+      typeof error.message === 'string'
```

Comment:
I don't understand, now all errors will be ignored? What problem is this fixing?

Reply:
![image](https://github.com/user-attachments/assets/dccbc64d-4982-4af2-8ba3-747691852145)
this error because is error.message was ignored.

Reply:
> but now not only this example error is ignored, but all errors. Which other errors are happening and why should they be ignored?

![image](https://github.com/user-attachments/assets/a3b068e5-2129-47ce-9af8-bd7ac319673d)

this is only thing I can trace back to.

I guess this just block error message in build